libs/react-components/community/src/components/form/file-upload/file-upload.stories.tsx

```diff
@@ -84,3 +84,12 @@ export const PdfAndTxtOnly: Story = {
     accept: '.pdf,.txt',
   },
 };
+
+export const SizeLimited: Story = {
```

Comment:
Example should show how the `onInvalidSize` fn can be used to notify user, when some files are not uploaded. But some are.

Reply:
Added an example.